graph/task_test.go

```diff
@@ -21,10 +21,16 @@ func TestUsingRegistryCreds(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		task := &Task{
+		var credentials []*Credential
+
+		credentials = append(credentials, &Credential{
 			RegistryName:     test.registry,
 			RegistryUsername: test.user,
 			RegistryPassword: test.pw,
+		})
+		task := &Task{
+			RegistryName: test.registry,
+			Credentials:  credentials,
```

Comment:
Simplified as `Credentials: []*Credential{ &Credential { ... } }` - no need for appends or variable. Same for below